tools/clang/include/clang/SPIRV/ModuleBuilder.h

```diff
@@ -54,6 +54,12 @@ class ModuleBuilder {
   inline void setAddressingModel(spv::AddressingModel);
   inline void setMemoryModel(spv::MemoryModel);
 
+  /// \brief Sets the Type of Shader (Vertex, Fragment, etc) in <theModule>
```

Comment:
No need to `<>` the Module. `<>` typically means some symbol that should typically be substituted by concrete values. Here `theModule` is one of the class variable, which is not some kind of symbolic representation.

Besides, comments for public methods are meant for users; so we should not involve implement implementation details in the comment. Just "Sets the shader kind (vertext, pixel/fragment, etc.) for the module under construction. We only support a single entry point per module for now." should be fine.